utils/new-design/all.py

```diff
@@ -0,0 +1,194 @@
+import os
+import yaml
+import itertools
+
+class ConfigVariants:
+    arch = ['arm64', 'x86_64']
```

Comment:
```suggestion
    arch = ['arm64', 'x86_64']
```

I think it would be better to have a dictionary here to keep track a number of options (an option means arch, hypervisor, platform etc)